pandas_access/__init__.py

```diff
@@ -34,7 +34,8 @@ def _extract_dtype(data_type):
     if data_type.startswith('double'):
         return np.float_
     elif data_type.startswith('long'):
-        return np.int_
+        # access CAN have null values on long type, @ pandas 0.24 int null suport is experimental, a float is safer for now.
```

Comment:
float supports a smaller range than INT. This might be ok, but when dealing with IDs (which tend to use up the whole range of values), you will break code.

consider: 

f = np.float_(1 << 53)
print(f)
f += 1
print(f)

(in case np.float_ is a 64 bit IEEE representation)

So this should at least be optional.

Reply:
Since pandas .24 there is a Int type (notice the caps on the I), which allows nulls ints. If you are willing to require .24+ (they are at .24.2 now), that will fix it for both cases.